tests/step-103.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+# Test:
+#   Fail on not available shared hooks.
+
+if ! sh /var/lib/githooks/install.sh; then
+    echo "! Failed to execute the install script"
+    exit 1
+fi
+
+mkdir -p /tmp/shared/hooks-103.git/pre-commit &&
+    echo 'exit 0' >/tmp/shared/hooks-103.git/pre-commit/succeed &&
+    cd /tmp/shared/hooks-103.git &&
+    git init &&
+    git add . &&
+    git commit -m 'Initial commit' ||
+    exit 1
+
+# Install shared hook url into a repo.
+mkdir -p /tmp/test103 && cd /tmp/test103 || exit 1
+git init || exit 1
+mkdir -p .githooks && echo '/tmp/shared/hooks-103.git' >.githooks/.shared || exit 1
+git add .githooks/.shared
+git hooks shared update
+
+RESULT=$(ls -A ~/.githooks/shared 2>/dev/null)
+if [ "$RESULT" = "" ]; then
+    echo "! Expected shared hooks to be installed."
+    exit 1
+fi
+git commit -m "Test" || exit 1
+
+# Remove all shared hooks
+git hooks shared purge || exit 1
+
+# Clone a new one
+echo "Cloning"
+cd /tmp || exit 1
+git clone /tmp/test103 test103-clone && cd test103-clone || exit 1
+
+# Remove all shared hooks
+git hooks shared purge || exit 1
```

Comment:
Can you please also add a test that fails on the double purge with the new config option set to true?